### PR TITLE
{math}[foss/2017b] gmsh v3.0.6

### PR DESCRIPTION
**Pull request by wpoely86**
(created using `eb --new-pr`)

edit: depends on ~~#5664~~ (`PETSc`) + #6260 (`SLEPc`)

### CODE REVIEW

**Comment by wpoely86** [comment]
Test report by @wpoely86
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
nic96 - Linux centos linux 7.4.1708, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz, Python 2.7.5
See https://gist.github.com/ff9e5155827e4d0b31de60307af43536 for a full test report.

**Comment by boegel** [comment]
@wpoely86 Missing PR for `SLEPc/3.8.3-foss-2017b-Python-2.7.14`?

cc @smoors

**Comment by smoors** [comment]
@wpoely86 @boegel see #6260

File: easybuild/easyconfigs/g/gmsh/gmsh-3.0.6-foss-2017b-Python-2.7.14.eb [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+builddependencies = [
+    ('CMake', '3.10.2'),
+    ('SWIG', '3.0.12', versionsuffix),
+]
+
+dependencies = [
+    ('Python', '2.7.14'),
+    ('PETSc', '3.8.3', '-downloaded-deps'),
+    ('SLEPc', '3.8.3', versionsuffix),
```
Comment (boegel) [inline easybuild/easyconfigs/g/gmsh/gmsh-3.0.6-foss-2017b-Python-2.7.14.eb]: @wpoely86 To use `SLEPc` from #6260, you'll need to drop the `versionsuffix` here...

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2411.golett.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/48dc6fd7c32aa16322d7a9df43a46f7a for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2065.delcatty.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/ac1d51259470fcfd0bd85948e1b79367 for a full test report.

**Comment by boegel** [comment]
Going in, thanks @wpoely86!

**Pull request merged**